extensions/vllm-semantic-router/src/manager/semantic-router-container-manager.ts

```
@@ -0,0 +1,591 @@
+
+  async init(): Promise<void> {
+    this.#factoryDisposable = this.provider.setSemanticRouterConnectionFactory({
+      type: 'vllm',
+      create: async (params: SemanticRouterCreateParams): Promise<SemanticRouter> => {
+        return this.create(params);
+      },
+    });
+  }
+
+  async create(params: SemanticRouterCreateParams): Promise<SemanticRouter> {
+    console.log(`Creating semantic router`, params);
```

Biome (2.5.6)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)